src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/vsts_cd_provider.py

```diff
@@ -0,0 +1,81 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=no-self-use,too-many-arguments,too-many-lines
```

Comment:
Please place the pylint disable at the places they need instead of at the file scope. This way you can avoid overskipping pylint checks.